%YAML:1.0
---

misc_dimension: !!opencv-camera_matrix
   rows: 4
   cols: 1
   dt: d
   data: [973, 1855.6400000000021, 1472.9599999999998, 3479.0500000000015]

car_dimension: !!opencv-camera_matrix
   rows: 4
   cols: 1
   dt: d
   data: [28742, 43862.69000000106, 46808.929999998705, 111632.61999999972]

van_dimension: !!opencv-camera_matrix
   rows: 4
   cols: 1
   dt: d
   data: [2914, 6430.010000000035, 5542.66, 14798.360000000066]

truck_dimension: !!opencv-camera_matrix
   rows: 4
   cols: 1
   dt: d
   data: [1094, 3557.3699999999817, 2828.0900000000174, 11059.330000000044]

pedestrian_dimension: !!opencv-camera_matrix
   rows: 4
   cols: 1
   dt: d
   data: [4487, 7900.290000000075, 2962.2700000000477, 3779.330000000002]

bicycle_dimension: !!opencv-camera_matrix
   rows: 4
   cols: 1
   dt: d
   data: [1627, 2826.429999999993, 970.950000000002, 2869.289999999997]